quran/models.py

```diff
@@ -0,0 +1,18 @@
+'''Models describing abstractions used at Quran app'''
+from django.db import migrations, models
+
+
+class Surah(models.Model):
+    '''General information about Surahs'''
+    name = models.CharField(max_length=20)
+    total_ayahs = models.PositiveIntegerField(default=0)
+
+
+class Ayah(models.Model):
+    '''General information about Ayah'''
+    ayah = models.PositiveIntegerField(default=0)
+    text = models.TextField()
```

Comment:
Define `set_text` and `get_text` methods, in order to think about this field as an array and don't bother yourself with its internal representation in the database.

Reply:
Can you define your setter and getter? That would be awesome. Look for example [here](https://stackoverflow.com/questions/22340258/django-list-field-in-model). But don't copy and paste.
Your setter should join your words with a comma, your getter should just split by comma. That's all.